Client/Frontend/Settings/Clearables.swift

```diff
@@ -126,24 +115,14 @@ class CacheClearable: Clearable {
             // Remove the basic cache.
             URLCache.shared.removeAllCachedResponses()
             
-            var err: Error?
-            for item in ["Caches", "Cookies", "WebKit"] {
-                do {
-                    try deleteLibraryFolderContents(item, validateClearedExceptFor: ["Snapshots"])
-                } catch {
-                    err = error
-                }
+            WKWebsiteDataStore.default().removeData(ofTypes: WKWebsiteDataStore.allWebsiteDataTypes(), modifiedSince: Date(timeIntervalSinceReferenceDate: 0)) {
```

Comment:
Previously we deleted only 3 items `["Caches", "Cookies", "WebKit"]`
Are we sure `allWebsiteDataTypes()` removes the same stuff and doesn't break anything?

Reply:
Yeah actually they used to delete more than required.

Reply:
question - is this tied to the 'clear cache' switch? if so, why does it also clear cookies?